tools/internal_ci/linux/grpc_coverage.sh

```diff
@@ -27,3 +27,10 @@ python tools/run_tests/run_tests.py \
   -c gcov                           \
   -x sponge_log.xml                 \
   -j 16
```

Comment:
I think we need to track whether this command fails, and exit the script appropriately to reflect whether the tests pass.